### ISSUE TITLE
Add contact page

### ISSUE THREAD

**Issue by theecrit** [openoakland/funding-public-safety#105]
## Description

There are a number of reasons a visitor might want to contact the FPS team but this isn't easily findable info.

### Potential solution

Add a contact page, either to the global nav or footer, that embeds the [current FPS interest form](https://bit.ly/fps-interest) and includes a link to the [bug report form](https://forms.gle/hKAEcA5AU5rxJz3i7).

**Comment by jaedoak** [comment]
I think that in order to embed the form (as opposed to just link out to it) I would need someone to pull code from the Google form editor (via Send > HTML ("<>" tab) > copy the snippet). Who has this access?

**Comment by jaedoak** [comment]
There's already a link to the form on the [Get Involved](https://openoakland.github.io/funding-public-safety/get-involved/) page. And there's going to be a link to the bug report form in a persistent "warning" header on every page (Issue 104). This contact page feels redundant. Is the idea just to collect all forms in one place for people who are just scanning the site for a "Contact" link?

Given that context, I'm leaning toward including it in the footer, in order to not clutter the header. I'm open to thoughts/context!

**Comment by theecrit** [comment]
@jaedoak Yeah, the idea is that the links get pretty buried and people who need it expect to find it quickly. Footer makes sense to me. 

As for the embed, I think you should have access. It lives in the shared drive (I think under Project Management > Engagement or something similar...I don't remember the subfolder). Lemme know if you can't find it.

**Comment by jaedoak** [comment]
Thanks @theecrit ! I need to build the instinct that I can probably find what I need in the drive. :)

I'll push a PR now for first draft of this implementation, which looks like this:

Contact Page:
<img width="967" alt="Screen Shot 2022-03-16 at 9 33 43 AM" src="https://user-images.githubusercontent.com/53203141/158642090-adfae842-fa03-494f-8622-79c0ab2357e5.png">

Footer, in web browser:
<img width="956" alt="Screen Shot 2022-03-16 at 9 35 40 AM" src="https://user-images.githubusercontent.com/53203141/158642131-d476c17f-b74e-4a5f-90c8-b4940948eab7.png">

Footer, in narrow web browser:
<img width="371" alt="Screen Shot 2022-03-16 at 9 35 56 AM" src="https://user-images.githubusercontent.com/53203141/158642194-e0d4c661-1068-4be6-bc73-c6d52731a842.png">
